jupyter-pytorch-full/tests/test_imports.py

```diff
@@ -13,14 +13,12 @@
 def main():
     """Test running container and imports."""
     rock = yaml.safe_load(Path("rockcraft.yaml").read_text())
-    name = rock["name"]
+    rock_image = rock["name"]
     rock_version = rock["version"]
```

Comment:
I think you could use chisme abstractions instead of doing this manually as we 're doing [here](https://github.com/canonical/kserve-rocks/blob/main/controller/tests/test_rock.py#L13-L15).

Reply:
done